File: zeno/classes/tag.py
from zeno.classes.base import CamelModel
from zeno.classes.slice import FilterIds


class Tag(CamelModel):
    tag_name: str
    folder: str
    selection_ids: FilterIds


class TagMetricKey(CamelModel):
    tag: Tag
    model: str
    metric: str

File: frontend/src/zenoservice/models/Tag.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { FilterIds } from "./FilterIds";

export type Tag = {
	tagName: string;
	folder: string;
	selectionIds: FilterIds;
};

export type TagMetricKey = {
	tag: Tag;
	model: string;
	metric: string;
};
